src/components/Header/User.tsx

```diff
@@ -22,6 +23,8 @@ export const User = ({ handleLogOut }: IHeaderUserProps) => {
     { href: '/profile/settings', label: t(Translation.NAVBAR_USER_MENU_EDIT_PROFILE) },
   ]
 
+  const isAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```

Comment:
fix name

Reply:
changed it to showMenuItems hope it's ok